SDL2_mixerConfig.cmake.in

```diff
@@ -82,7 +82,11 @@ include(CMakeFindDependencyMacro)
     endif()
 
     if(SDL2MIXER_VORBIS_VORBISFILE AND NOT SDL2MIXER_VENDORED AND NOT TARGET vorbisfile::vorbisfile)
```

Comment:
Since the package is `Vorbis`, shouldn't the target checked be `Vorbis::vorbisfile` ?

Reply:
You're right. This was the last `Vorbis::vorbisfile`.

Reply:
Fixed in 6825230